pkg/proxy/iptables/proxier_test.go

```diff
@@ -1307,18 +1308,33 @@ func Test_sortIPTablesRules(t *testing.T) {
 	}
 }
 
+// getLine returns the line number of the caller, if possible.  This is useful in
+// tests with a large number of cases - when something goes wrong you can find
+// which case more easily.
+func getLine() int {
```

Comment:
@aojea recently pointed me to [t.Helper()](https://pkg.go.dev/testing#T.Helper), which if you call it in a function, it causes future `t.Errorf()` etc calls in that function to print line numbers as though they had been called from the parent function instead...

Reply:
Righ, not so helpful with a table-test.  I used this line-num trick in REST tests, it helped a TON.